File: Java/백준/Bronze/10757. 큰 수 A＋B/큰 수 A＋B.java
import java.math.BigInteger;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        // A와 B 입력받기
        String A = sc.next();
        String B = sc.next();

        // BigInteger로 변환하여 덧셈
        BigInteger numA = new BigInteger(A);
        BigInteger numB = new BigInteger(B);
        BigInteger result = numA.add(numB);

        // 결과 출력
        System.out.println(result);

        sc.close();
    }
}

File: Kotlin (JVM)/백준/Bronze/10757. 큰 수 A＋B/큰 수 A＋B.kt
import java.math.BigInteger

fun main() {
    // 입력받기
    val (A, B) = readLine()!!.split(" ")

    // BigInteger로 변환 후 덧셈
    val result = BigInteger(A) + BigInteger(B)

    // 결과 출력
    println(result)
}